### DISCUSSION TITLE
Github app does not support translation

### DISCUSSION THREAD

**Discussion by Liu-Jinshuai** [community/community#75712] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I installed the github app on my mobile phone. When I search for a git project on the website and want to view the details, it always automatically opens the github app, but the app does not provide a translation function, which makes people feel very uncomfortable. It seems that you can only uninstall it temporarily, otherwise it will keep jumping. I hope to provide a translation function for the app. Thanks.

**Comment by ghostinhershell** [comment]
Hey there @Liu-Jinshuai, 

Thanks so much for reaching out in our community. Can you please share a bit more information about the issue you're running into here? For example, is this a [localization](https://www.google.com/search?q=localization+meaning+apps&sca_esv=584647371&rlz=1C5GCEM_enUS1079US1079&ei=9FBeZbqmG4i2qtsP4Oij8Ao&ved=0ahUKEwi68Je2ptiCAxUIm2oFHWD0CK4Q4dUDCBA&uact=5&oq=localization+meaning+apps&gs_lp=Egxnd3Mtd2l6LXNlcnAiGWxvY2FsaXphdGlvbiBtZWFuaW5nIGFwcHMyBhAAGBYYHjIIEAAYFhgeGA8yBhAAGBYYHjIGEAAYFhgeMggQABgWGB4YDzIIEAAYFhgeGA8yCBAAGBYYHhgPMgYQABgWGB4yBhAAGBYYHjILEAAYgAQYigUYhgNIyQhQ-AJYsgdwAHgCkAEAmAGmAqABvAaqAQUwLjIuMrgBA8gBAPgBAcICBBAAGEfCAgsQABiABBiKBRiRAsICBRAAGIAE4gMEGAAgQYgGAZAGCA&sclient=gws-wiz-serp) issue where you're finding that the GitHub app doesn't translate to your local language? Just want to make sure that I'm understanding your issue properly.

**Reply by Liu-Jinshuai** [reply]
Thank you for your reply. For example, if I use the github app to view the README.md document of a certain project (not the browser, the browser supports translation), can the translation function be provided in the app?

**Reply by Liu-Jinshuai** [reply]
The app I’m talking about here refers to the github app downloaded from a mobile application store (such as the App Store).

**Comment by 100Khushi** [comment]
I think exploring external [audio translation](https://wavel.ai/solutions/ai-video-translator) tools might help bridge the gap. These tools can assist in translating text or content into various languages, potentially making your experience more accessible.

**Reply by Liu-Jinshuai** [reply]
Thank you for your reply. In fact, sometimes I want a translation function when viewing the README.md document in the github application on my mobile phone (preferably other code pages also support translation)
